src/python/pants/backend/go/subsystems/golang.py

```diff
@@ -90,6 +90,12 @@ class EnvironmentAware(Subsystem.EnvironmentAware):
             help="Name of the tool to use to compile fortran code included via CGo in a Go package.",
         )
 
+        external_linker_binary_name = StrOption(
+            default="gcc",
+            advanced=True,
+            help='Name of the tool to use as the "external linker" when invoking `go tool link`.',
```

Comment:
Does this need to be on the `[golang].go_search_paths `?

Reply:
Ah ha, I was looing at wrong version of the docs. It'd be helpful to mention `[golang].cgo_tool_search_paths` here

Reply:
Done.